### DISCUSSION TITLE
How to quickly delete a bucket with large number of files?

### DISCUSSION THREAD

**Discussion by mshanmu** [minio/minio#16772] [Q&A] [answered]
It looks like it takes a very long time to delete a bucket with large number of files in it. Doing "mc rb" or "mc rm --force --recursive" is taking forever. 

So, Is there a truncate command to quickly delete a bucket?

**Comment by harshavardhana** [comment, marked as answer]
Deletes are as good as your drives. They will take time if there are millions.
